chatbot_core/cli.py

```diff
@@ -74,6 +76,7 @@ def cli_start_mq_bot():
                         help="Chatbot entrypoint name", type=str)
     args = parser.parse_args()
     run_mq_bot(args.bot_name)
+    Event().wait()
```

Comment:
Probably better to use `wait_for_exit_signal` from `ovos-utils` to explicitly continue on `SIGINT`. Is there something we should do to cleanly exit/shutdown the bot?

Reply:
> Probably better to use `wait_for_exit_signal` from `ovos-utils` to explicitly continue on `SIGINT`. Is there something we should do to cleanly exit/shutdown the bot?

I do not think it is technically possible, at least implementation of such mechanism probably would not worth it at the moment